volumes/csi-wrapper/hack/ibm/nginx-kata-with-my-pvc-and-csi-wrapper.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       envFrom:
         - configMapRef:
             name: ibm-vpc-block-csi-configmap
-      image: gcr.io/k8s-staging-cloud-provider-ibm/ibm-vpc-block-csi-driver:v4.3.1
+      image: gcr.io/k8s-staging-cloud-provider-ibm/ibm-vpc-block-csi-driver:master
```

Comment:
Shall we use a fixed version to avoid broken? Like v5.x if it's the latest tag right now?

Reply:
It should be fine to use master branch at here and we do not need update driver tag when there is a new release is ready to use. When the driver release a master build, we can assume that it must already be verified. And if there are any broken, we can check related logs to identify the real root cause easily.

Reply:
I think that's exactly what I'm concerned. We'd better keep a bunch of document and scripts that can be verified at any time. Using `master` branch means it can be broken very easily, example is v4.3.1 vs current `master` branch.

Reply:
got it, update `master` to latest `v5.2.0` to avoid broken.